tests/phpunit/tests/import/import.php

```diff
@@ -19,11 +24,7 @@ public function set_up() {
 
 		add_filter( 'import_allow_create_users', '__return_true' );
 
-		if ( ! file_exists( DIR_TESTDATA . '/plugins/wordpress-importer/wordpress-importer.php' ) ) {
-			$this->fail( 'This test requires the WordPress Importer plugin to be installed in the test suite. See: https://make.wordpress.org/core/handbook/contribute/git/#unit-tests' );
-		}
-
-		require_once DIR_TESTDATA . '/plugins/wordpress-importer/wordpress-importer.php';
+		require_once $importer;
```

Comment:
>Now, I can imagine the choice to mark the tests as "failed" may have been to prevent contributors missing the message about the need to install the Importer plugin.

Looking at the history, skip was previously used but then intentionally converted to `fail` in [Trac 40533](https://core.trac.wordpress.org/ticket/40533) via changesets [r40531](https://core.trac.wordpress.org/changeset/40531) and [r40532](https://core.trac.wordpress.org/changeset/40532). [This comment](https://core.trac.wordpress.org/ticket/41345#comment:3) has more of the reasoning:

>I don't think we should skip these tests - the changes in [[40531]](https://core.trac.wordpress.org/changeset/40531) and [[40532]](https://core.trac.wordpress.org/changeset/40532) were intentional. Without those changes, if someone has an incomplete test environment and they write patches against plugins or another feature that is left out of their local testing, then they're increasing the chance of writing a bad patch that fails a build later on. Quoting from [#40533](https://core.trac.wordpress.org/ticket/40533):
>>If the test data isn't valid or the environment is not capable of supporting the test then the test should fail, otherwise the tests are not reliable.

Given the hard fail is intentional to "prevent contributors missing the message",  thinking @jrfnl suggestion achieves the goals:
>If that's the case, I'd suggest adding a file_exists() check for the importer plugin to the test bootstrap.php file and exiting the test run if that prerequisite is not fulfilled, along the same lines as the check that the database connection can be made and that the correct version of the Polyfills is installed.

@jrfnl do you agree?

Reply:
I agree with you @jrfnl.

Hey @johnbillion what do you think? Are you okay with setting a hard requirement in the bootstrap rather than fail the tests?